src/ui/components/DevToolsProcessingScreen.tsx

```diff
@@ -36,7 +36,7 @@ export function DevToolsProcessingScreen() {
 
   if (showStalledMessage) {
     secondaryMessage = "There may be a problem. This is taking much longer than expected.";
-  } else if (recording?.duration != null) {
+  } else if (recording?.duration) {
```

Comment:
So for `recording.duration === 0` we'd show now: "This could take a while, depending on the complexity and length of the replay.", right? Would it make sense to display a prediction for `1` instead of that? Are we talking about duration in seconds here?

Reply:
I'm chatting with Miriam right now about a user report of a 20 minute recording that (for some reason) has duration:0 in the DB. I think it's safest not to estimate anything in that case. (We _were_ estimating 1 minute, but that was way off.)

Reply:
`formatEstimatedProcessingDuration` rounds up to the nearest min for what it's worth, so if we did nothing (current behavior) we'd predict 1 minute.